README.md

```diff
@@ -1,6 +1,8 @@
 Mod Public License, v2. WIP.
 ======================================
-This license is intended to fix some of the problems and loopholes in the original [MMPL](https://github.com/BuildCraft/BuildCraft/blob/6.1.x/buildcraft_resources/LICENSE.txt), and in general make a modder-friendly, easily usable Open Source license for mods of all types.
+This license is intended to fix some of the problems and loopholes in the original [MMPL](https://github.com/BuildCraft/BuildCraft/blob/6.1.x/buildcraft_resources/LICENSE.txt), and in general make a modder-friendly, easily usable permissive license for mods of all types.  
+
+The goal is to achieve a compromise between protecting modders from theft and benefitting the community by allowing them to work with the code and create and distribute their own derivatives. Unfortunately the first intent is incompatible with the idea of Open Source, but we want to make it reasonably close to Open Source.

```

Comment:
Your continued insistance that this "isn't an Open Source license" is starting to get on my nerves. We all get that you hold your own definition of what "Open Source" is and isn't. I've stated many many many times that this is a work in progress. Your refusal to respect our differences in opinion is impeding your ability to successfully contribute to this.

Stop it already.